### PR TITLE
Custom Tap Highlight Color

### PR DESCRIPTION
**Pull request by nirzar**
https://phabricator.wikimedia.org/T93173

Custom tap highlight color for cells on feed. It uses category based
off UItableViewCell

### CODE REVIEW

File: Wikipedia/Categories/UITableViewCell+SelectedBackground.m [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//  Created by Monte Hurd on 12/13/14.
+//  Copyright (c) 2014 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import "UITableViewCell+SelectedBackground.h"
+
+@implementation UITableViewCell (SelectedBackground)
+
+- (void)SetSelectedBackground {
```
Comment (bgerstle) [inline Wikipedia/Categories/UITableViewCell+SelectedBackground.m]: @nirzar naming convention for category methods is: `[wmf_]camelCaseMethodName`.  Categories on classes outside our app have a `wmf_` prefix, and all ObjC methods are camel case.  The prefix prevents potential clashing issues, which can happen as a result of Objective-C's lack of module namespaces in the runtime.  The Swift runtime doesn't have this issue.

File: Wikipedia/Categories/UITableViewCell+SelectedBackground.h [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//  Created by Monte Hurd on 12/13/14.
+//  Copyright (c) 2014 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
```
Comment (montehurd) [inline Wikipedia/Categories/UITableViewCell+SelectedBackground.h]: I did? ;)

File: Wikipedia/Categories/UITableViewCell+SelectedBackground.h [hunk 3]
```diff
@@ -0,0 +1,10 @@
+//  Created by Monte Hurd on 12/13/14.
```
Comment (montehurd) [inline Wikipedia/Categories/UITableViewCell+SelectedBackground.h]: I did? ;)

File: Wikipedia/Categories/UIColor+WMFStyle.m [hunk 4]
```diff
@@ -167,6 +167,16 @@ + (instancetype)wmf_blueTintColor {
     return c;
 }
 
++ (instancetype)wmf_tapHighlightColor {
+    static UIColor* c = nil;
+    
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        c = [UIColor colorWithRed:238.0f/255.0f green:238.0f/255.0f blue:238.0f/255.0f alpha:1];
```
Comment (montehurd) [inline Wikipedia/Categories/UIColor+WMFStyle.m]: See line 191 if you want to just use hex for the color.

**Comment by montehurd** [comment]
Nirzar has one more commit, I think, on the way.

**Pull request merged**